packages/quantification/src/net-quantification.ts

```diff
@@ -0,0 +1,185 @@
+import type {
+  AnnualTotals,
+  UnadjustedQuantificationSummary,
+} from './quantification';
+
+/**
+ * Calculates the net tonnes removed accounting for years with emissions. Years with emissions (represented
+ * by a negative number) should be reduced to zero and any tonnes removed should be reduced by the emitted
+ * amount.
+ *
+ * Algorithm overview:
```

Comment:
This is a super helpful overview and helped me answer my own question above. Great documentation!

Reply:
Thanks!